_posts/2020-03-05-snowplow-android-tracker-1.4.0-released/index.md

```diff
@@ -0,0 +1,169 @@
+
+Read on below for:
+
+1. [Tracking GDPR basis for processing with the GDPR context](#gdpr)
+2. [Diagnostic feature for internal error tracking](#diagnostic)
+3. [Fix for IAB enrichment identification](#iab)
+4. [Make POST path of Emitter configurable](#custompath)
+5. [Updates and bug fixes](#updates)
+6. [Documentation](#documentation)
+7. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="gdpr">1. Tracking GDPR basis for processing with the GDPR context</h2>
+
+This release introduces the `gdprContext` and the `enableGdprContext` methods, which append a GDPR context to all events once enabled ([#312][312]).
+This allows users to easily record the basis for data collection and relevant documentation, and enables a straightforward audit flow for all events.
+
+It takes the following arguments:
+
+|      **Name** | **Description**             | **Required?** | **Type** |
+|--------------:|:----------------------------|:--------------|:---------|
+|`basisForProcessing` | GDPR Basis for processing | Yes           | Enum String   |
+|     `documentId` | ID of a GDPR basis document     | No           | String   |
+|        `documentVersion` | Version of the document        | No            | String   |
+| `documentDescription` | Description of the document | No            | String   |
+
+The required basisForProcessing accepts only the following literals: `consent`, `contract`, `legal_obligation`, `vital_interests`, `public_task`, `legitimate_interests` - in accordance with the [five legal bases for processing](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/)
+
+The GDPR context is enabled by calling the `gdprContext` method of the tracker builder or by calling the `enableGdprContext` method once the tracker has been initialised.
+
+It is called as follows:
+
+{% highlight java %}
+Tracker.instance().enableGdprContext(
+    Gdpr.Basis.CONSENT,
+    "someId",
+    "0.1.0",
+    "a demo document description"
+);
+{% endhighlight %}
+
+
+<h2 id="diagnostic">2. Diagnostic feature for internal error tracking</h2>
+
+The tracker is constantly tested with different platforms and versions but there are rare situations where an error can be reported. The tracker manages errors internally, avoiding crashes of the app and assuring that the events are not lost. However, it can now report errors to a Snowplow collector as `diagnostic_error` events.
+
+To activate this feature you only need to enable `trackerDiagnostic` ([#343][343]):
+
+{% highlight java %}
+TrackerBuilder trackerBuilder =
+    new TrackerBuilder(emitter, namespace, appId, appContext)
+        .trackerDiagnostic(true)
+        ...
+        .build();
+Tracker.init(trackerBuilder);
+{% endhighlight %}
+
+It will automatically report any internal tracker error to the configured Snowplow collector. These will be fundamental for the troubleshooting unexpected behaviour of the tracker.
```

Comment:
```suggestion
It will automatically report any internal tracker errors to the configured Snowplow collector. These will be fundamental for troubleshooting unexpected behaviour of the tracker.
```